src/main/java/gov/usgs/cida/pubs/busservice/sipp/SippProcess.java

```diff
@@ -131,6 +137,7 @@ protected InformationProduct getInformationProduct(String ipNumber) {
 	}
 
 	protected PublicationSeries getSeriesTitle(PublicationSubtype pubSubtype, String text) {
+		PublicationSeries ret = null;
```

Comment:
"ret" is not the best variable name. maybe pubSeriesTitle or something describing what is returned.